tutorials/FITS-header/FITS-header.ipynb

```diff
@@ -18,187 +20,288 @@
+   "metadata": {
+    "id": "ml2DVqgRlJwQ"
+   },
    "source": [
-    "Now let's change the header to give it the correct object:"
+    "There are alot of other useful commands to remember for reading FITS files:"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "LXnCQLIqltzv"
+   },
    "source": [
-    "header['OBJECT'] = \"M31\""
+    "1. `fits.open` allows us to look more generally at our data."
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "id": "tbmPXHehmO8q"
+   },
    "source": [
-    "Finally, we have to write out the FITS file. Again, the convenience \n",
-    "function for this is the most useful command to remember:"
+    "2. `fits.getheader` will give you the same output as `fits_inf[0].header`. By changing the HDU number to `fits_inf[1]header`, what will the output be? "
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "id": "YKrZaxbRmv2a"
+   },
    "outputs": [],
    "source": [
-    "fits.writeto('output_file.fits', data, header, overwrite=True)"
+    "# But hdu_number = 0 is the PRIMARY HDU. How many HDUs are in this file?\n",
+    "fits_inf = fits.open(\"input_file.fits\")\n",
+    "fits_inf.info() \n",
+    "fits_inf[0].header"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "id": "z_ddsyRUoVu8"
+   },
    "source": [
-    "That's it; you're done!"
+    "**Thought Question:** *How many HDUS are in this file? Were you right about the number of cards in the primary HDU?*"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "id": "AKTWFAyeopR4"
+   },
    "source": [
-    "Two common and more complicated cases are worth mentioning (but if your needs \n",
-    "are much more complex, you should consult the full documentation http://docs.astropy.org/en/stable/io/fits/). \n",
-    "\n",
-    "The first complication is that the FITS file you're examining and \n",
-    "editing might have multiple HDU's (extensions), in which case you can \n",
-    "specify the extension like this:"
+    "**Thought Question:** *What will happen if you type* `fits_inf[2].header`*?*"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "UFYUH450mv2b"
+   },
+   "source": [
+    "Now that we know how to read out FITS files, let's change the header. The header is always changed via it’s keyword. The keywords are the 8 letter word to the left when you extract the header info. Lets change change the header to give the correct OBJECT:"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "id": "LOP18OR0mv2b"
+   },
    "outputs": [],
    "source": [
-    "data1, header1 = fits.getdata(\"input_file.fits\", ext=1, header=True)"
+    "header['OBJECT'] = \"M31\""
```

Comment:
I would add a second example here showing that you can use lower case, maybe something like this

```python
print(header['object'])
```

along with some text explaining that the keywords are not case sensitive.

Reply:
will do! thanks